### PR TITLE
Add Selenium WebDriver Configuration

### LINKED ISSUE #29: Add Selenium WebDriver configuration

### PR DESCRIPTION
**Pull request by NadejdaClima**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,7 +39,17 @@
             <version>6.6.0</version>
```
Comment (dceaicovschi) [inline pom.xml]: please check the duplicate of dependence

Comment (NadejdaClima) [reply pom.xml]: there are no duplicates
one dependency is cucumber-java, and the other is cucumber-junit

Comment (dceaicovschi) [reply pom.xml]: sorry, my mistake

**Review by MarcelMorozniuc — APPROVED** [review]

**Review by margaritaq — APPROVED** [review]

File: src/main/java/WebDriversManager.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
```
Comment (EugenS21) [inline src/main/java/WebDriversManager.java]: try to refactir it as singletone

**Review by EugenS21 — CHANGES_REQUESTED** [review]

File: src/main/java/WebDriversManager.java (hunk 2 above)
Comment (NadejdaClima) [reply src/main/java/WebDriversManager.java]: Done, please advise if changes are correct

**Pull request merged**